lib/pbench/test/unit/agent/task/test_results_push.py

```diff
@@ -17,26 +18,18 @@ class TestResultsPush:
     URL = "http://pbench.example.com/api/v1"
 
     @staticmethod
-    def add_success_mock_response():
+    def add_http_mock_response(code: int = 200):
```

Comment:
Are we happy with the numeric literal for the HTTP status code here (and at line 173), or would we rather we used a symbolic reference?